### DISCUSSION TITLE
Alphabetical sort of packages (for an organization)

### DISCUSSION THREAD

**Discussion by hansfn** [community/community#50697] [Packages]
### Select Topic Area

Product Feedback

### Body

On https://github.com/orgs/ORGNAME/packages the only "sort by" options are most and least downloads. That isn't very helpful if you want to get an overview of all the packages based on name.

This seems to have been fixed for repositories where you can sort after last updated, name or stars.

**Comment by marcotrosi** [comment]
I agree, and I found this discussion because I was searching for the same but for repositories sorting.
The feature has been added some time ago as you also noticed, but it seems it was forgotten to implement a configurable default value for sorting in the organization settings.
Maybe you want to be able to configure also the default sorting for packages?
--marco
